orochi/client.py

```diff
@@ -685,6 +705,12 @@ def do_status(self, s=''):
             parts.append('(%s)' % track_year)
         print(' '.join(parts) + '.')
 
+        # Log the current song to a file (can be used by other programs)
+        if self._log_current_song is True:
+            f = open(expanduser("~") + '/.orochi.current_song.log', 'w')
```

Comment:
Instead of logging to `~/.orochi.current_song.log` (an arbitrary path) you should adhere to the [XDG basedir specification](http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html) and store the file in `$XDG_CACHE_HOME/orochi/current_song.txt`.

You don't need to add that logic yourself though, instead just use the following:

```python
cachedir = get_orochi_xdg_dir('XDG_CACHE_HOME', '.cache')
filename = os.path.join(cachedir, 'current_song.txt')
```

Then, instead of assigning a file object, you can use a context manager instead:

```python
with open(filename, 'w') as songfile:
    songfile.write(...)
```

Then you don't need to explicitly close the file and can make sure that the file is always closed, even if an error occurs.